### PR TITLE
Electronics

### PR DESCRIPTION
**Pull request by dakoko**
Added electronics and firmware documentation.

### CODE REVIEW

File: electronics/README.asciidoc [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# OWO mainboard documentation
+
+## General information
+
+### Power
+The mainboard is split into 2 isolated sides with seperate ground planes.
+The microcontroller side is powered with 5V by an USB connector that is stepped down to 3.3V by a 5V to 3V voltage regulator (NCP164CSN330T1G). 
+The motor driver side is powered by a 4S LIPO battery (up to 16.8V) and is stepped down to 5V to power the drivers, isolators and servos
```
Comment (ReikoR) [inline electronics/README.asciidoc:8]: > 5V to power the drivers

Motor drivers are powered from battery.

File: electronics/README.asciidoc [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# OWO mainboard documentation
+
+## General information
```
Comment (ReikoR) [inline electronics/README.asciidoc:3]: Specify motor driver output current limit.

File: electronics/README.asciidoc [hunk 3]
```diff
@@ -0,0 +1,71 @@
+# OWO mainboard documentation
+
+## General information
+
+### Power
+The mainboard is split into 2 isolated sides with seperate ground planes.
+The microcontroller side is powered with 5V by an USB connector that is stepped down to 3.3V by a 5V to 3V voltage regulator (NCP164CSN330T1G).
```
Comment (ReikoR) [inline electronics/README.asciidoc:7]: > 5V to 3V voltage regulator

Voltage regulator has 3.3V output.

**Review by ReikoR — CHANGES_REQUESTED** [review]

File: electronics/README.asciidoc (hunk 1 above)
Comment (dakoko) [reply electronics/README.asciidoc]: Fixed

File: electronics/README.asciidoc (hunk 2 above)
Comment (dakoko) [reply electronics/README.asciidoc:3]: Fixed

File: electronics/README.asciidoc (hunk 3 above)
Comment (dakoko) [reply electronics/README.asciidoc]: Fixed

File: electronics/README.asciidoc [hunk 4]
```diff
@@ -4,10 +4,10 @@
 
 ### Power
 The mainboard is split into 2 isolated sides with seperate ground planes.
-The microcontroller side is powered with 5V by an USB connector that is stepped down to 3.3V by a 5V to 3V voltage regulator (NCP164CSN330T1G). 
-The motor driver side is powered by a 4S LIPO battery (up to 16.8V) and is stepped down to 5V to power the drivers, isolators and servos 
+The microcontroller side is powered with 5V by an USB connector that is stepped down to 3.3V by a 5V to 3.3V voltage regulator (NCP164CSN330T1G). 
+The motor driver side is powered by a 4S LIPO battery (up to 16.8V) and is stepped down to 5V to power the isolators and servos 
 using a buck converter (TPS62933DRLR). Both the buck converter and voltage regulator have power LEDs.
-The board can power 3 brushed DC motors, its encoders, 2 servos and one external IR sensor and IR LED.
+The board can power 3 brushed DC motors with up to 2.95A each, its encoders, 2 servos and one external IR sensor and IR LED.
```
Comment (ReikoR) [inline electronics/README.asciidoc:10]: How did you calculate 2.95A? I think it's slightly different.

Comment (dakoko) [reply electronics/README.asciidoc:10]: Using the datasheet formula:
Vitrip = 2.64V (as set by the 100k resistor on itrip)
Ripropi = 2.7k ohm
Aipropi = 3070 ( i used the wrong constant here)
ITRIP regulation level = VITRIP / RIPROPI X AIPROPI
The problem is now fixed

**Review by ReikoR — APPROVED** [review]

**Pull request merged**